api/src/test/java/uk/gov/hmcts/payment/api/controllers/PaymentControllerTest.java

```diff
@@ -708,15 +702,28 @@ public void testEnableFeatureFlag_withCorrectUID() throws Exception {
             .post("/api/ff4j/store/features/payment-search/enable")
             .andExpect(status().isAccepted());
 
-        MvcResult result = restActions
-            .get("/api/ff4j/store/features/payment-search")
-            .andExpect(status().isOk())
-            .andReturn();
+        Feature feature = ff4j.getFeature("payment-search");
+        assertThat(feature.getUid()).isEqualTo("payment-search");
+        assertThat(feature.isEnable()).isEqualTo(true);
+        assertThat(feature.getDescription()).isEqualTo("Payments search API");
+    }
 
-        FeatureApiBean feature = objectMapper.readValue(result.getResponse().getContentAsByteArray(), FeatureApiBean.class);
+    @Test
+    @Transactional
+    public void testDisableFeatureFlag_withCorrectUID() throws Exception {
+        restActions
+            .post("/api/ff4j/store/features/payment-search/disable")
+            .andExpect(status().isAccepted());
+
+        Feature feature = ff4j.getFeature("payment-search");
         assertThat(feature.getUid()).isEqualTo("payment-search");
-        assertThat(feature.getEnable()).isEqualTo(true);
+        assertThat(feature.isEnable()).isEqualTo(false);
```

Comment:
I would strongly suggest to use `isFalse()`, rather than `isEqualTo(false)`. The same to other places where you use `isEqualTo(true or false)` replace with `isTrue()` or `isFalse()`